### ISSUE TITLE
too many open files, uv_cwd

### ISSUE THREAD

**Issue by ar-naturalint** [Bogdan-Lyashenko/codecrumbs#24]
```bash
Client connected.
path.js:1084
          cwd = process.cwd();
                        ^

Error: EMFILE: too many open files, uv_cwd
    at Object.resolve (path.js:1084:25)
```

**Comment by Bogdan-Lyashenko** [comment]
How many files do you have there?)

**Comment by Bogdan-Lyashenko** [comment]
@ar-naturalint I just published v1.1.1, it excludes `node_modules` (and few more fixes).
Can you please test and let me know it's fine now? If it's still failing I will think about chunking approach with reading files. I will close this issue now, there is a duplicate https://github.com/Bogdan-Lyashenko/codecrumbs/issues/29 with a bigger discussion for you to follow. Thanks.
